.gitignore

```diff
@@ -1,5 +1,10 @@
+*.bz2
 *.csv
 *.csv.gz
+*.csv.zip
+*.tar.gz
+*.zip
```

Comment:
The `rapids-optuna-hpo` example ends up downloading a `.zip` file, which contains some inner zipped CSV files.

That's why I added `.zip` and `.csv.zip` here, and then though "might as well as some other common archives too", as I don't think we ever want any of these in source control.